commands/review_apps/disable.js

```diff
@@ -0,0 +1,77 @@
+const cli = require('heroku-cli-util')
+const co = require('co')
+const api = require('../../lib/api')
+const KolkrabbiAPI = require('../../lib/kolkrabbi-api')
+
+module.exports = {
+  topic: 'reviewapps',
+  command: 'disable',
+  description: 'disable review apps or settings on an existing pipeline',
+  help: `Example:
+
+    $ heroku reviewapps:disable -p mypipeline -a myapp --autodeploy
+    Disabling auto deployment ...
+    Configuring pipeline... done`,
+  needsApp: false,
+  needsAuth: true,
+  wantsOrg: false,
+  args: [],
+  flags: [
+    {
+      name: 'pipeline',
+      char: 'p',
+      description: 'name of pipeline',
+      hasValue: true
+    },
+    {
+      name: 'app',
+      char: 'a',
+      description: 'parent app used by review apps',
+      hasValue: true,
+      required: true
+    },
+    {
+      name: 'autodeploy',
+      char: 'p',
+      description: 'autodeploy the review app',
+      hasValue: false
+    },
+    {
+      name: 'autodestroy',
+      char: 'u',
```

Comment:
Same as with `-p` (`--autodeploy`). `--autodestroy` should be good I think.

Reply:
Updated.